{"name": "GET Non-Existent User", "status": "passed", "description": "\n    Test retrieving a non-existent user.\n    Expects a 404 Not Found response.\n    ", "attachments": [{"name": "stdout", "source": "e88fe238-f763-4c9c-997c-39d1f6feb787-attachment.txt", "type": "text/plain"}], "start": 1751563881160, "stop": 1751563881541, "uuid": "665a09a8-80bd-4596-ac08-26f34987c52d", "historyId": "314fdc1812f35ca4a065312d1edd7e33", "testCaseId": "314fdc1812f35ca4a065312d1edd7e33", "fullName": "tests.test_users_api#test_get_user_not_found", "labels": [{"name": "severity", "value": "minor"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_users_api"}, {"name": "host", "value": "codespaces-2886b5"}, {"name": "thread", "value": "24545-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_users_api"}]}